packages/css-library/src/utilities/border.scss

```diff
@@ -7,7 +7,7 @@ $u-border-base: (
   border: (
     base: 'vads-u-border',
     modifiers: (
-      noModifier: '-',
+      noModifier: '',
```

Comment:
A small bug fix. When it was set as `noModifier: '-',`, some border utilities were getting generated like this:

```css
  .mobile\:vads-u-border--7px {
    border-: 7px solid;
  }
```
Instead of this:

```css
  .mobile\:vads-u-border--7px {
    border: 7px solid;
  }
```